gnuradio-runtime/include/gnuradio/memory.h

```diff
@@ -0,0 +1,33 @@
+/* -*- c++ -*- */
+/*
+ * Copyright 2020 Free Software Foundation, Inc.
+ *
+ * This file is part of GNU Radio
+ *
+ * SPDX-License-Identifier: GPL-3.0-or-later
+ *
+ */
+
+#ifndef INCLUDED_GR_MEMORY_H
+#define INCLUDED_GR_MEMORY_H
+
+namespace gr {
```

Comment:
If we do this, we could also inject it into the `std` namespace.

Reply:
> It is undefined behavior to add declarations or definitions to namespace std or to any namespace nested within std

— https://en.cppreference.com/w/cpp/language/extending_std

So I wouldn't do that.

Reply:
I'm in agreement with Thomas here. Let's leave this in `gr::`.

Reply:
I don't think this really needs a maintainer guideline, but in case anyone asks later: nope, GR never inserts into the `std` namespace (or really, any namespace it shouldn't "own").